libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -293,6 +297,9 @@ public void setAccountSwitcherActivityClass(Class<? extends AccountSwitcherActiv
      * @param name The name associated with the key.
      * @return The key used for encrypting salts and keys.
      */
+    /*
+     * TODO: Mark this deprecated and remove it in Mobile SDK 7.0.
+     */
     public String getKey(String name) {
```

Comment:
Looks like my comment was lost. I didn't want to mark them deprecated in `6.0` because this may trigger app developers to remove their implementation from `6.0`, leaving us with no upgrade path. I was gonna mark it deprecated in `6.1` instead.